srcpkgs/swapspace/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'swapspace'
+pkgname=swapspace
+version=1.17
+revision=1
+wrksrc=Swapspace-${version}
```

Comment:
`pkgname=Swapspace` as Void uses tarball/dirname/repo name. Then no need to define wrksrc

Reply:
Don't we want a lowercase pkgname? I assumed `Swapspace` is just a quirk in repo naming.

Reply:
There has been a discussion about switching to lowercase only package names, but it would require changes in xbps and some people were against it, so we are staying with the old policy - respect how upstream names it.

Reply:
Alright then, done